src/pages/eth2/staking-address.js

```diff
@@ -0,0 +1,347 @@
+  border-bottom: 1px solid ${(props) => props.theme.colors.border};
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    flex-direction: column;
+  }
+`
+
+const LeftColumn = styled.div`
+  flex: 1 1 50%;
+  padding: 2rem;
+  padding-top: 5rem;
+`
+
+const RightColumn = styled(LeftColumn)`
+  background: ${(props) => props.theme.colors.ednBackground};
+`
+
+const Title = styled.h1`
+  font-weight: normal;
+  font-size: 2rem;
+  line-height: 140%;
+  color: ${(props) => props.theme.colors.text};
+`
+
+const Subtitle = styled.div`
+  font-size: 20px;
+  line-height: 140%;
+  color: ${(props) => props.theme.colors.text200};
+  margin-bottom: 3.5rem;
+`
+
+const ButtonRow = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const StyledButton = styled(ButtonLink)`
+  margin-top: 0rem;
+`
+
+const StyledLink = styled(Link)`
+  margin-left: 1rem;
+`
+
+const StyledCard = styled(Card)`
+  margin-bottom: 3rem;
+`
+
+const DumbTag = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 8px 8px;
+  width: 100%;
+  margin-bottom: 0.5rem;
+  margin-right: 0.5rem;
+  background: ${(props) => props.theme.colors.cardGradient};
+  border-bottom: 1px solid ${(props) => props.theme.colors.border};
+  color: ${(props) => props.theme.colors.text};
+  border-radius: 3px 3px 0px 0px;
+  text-transform: uppercase;
+  font-size: 14px;
+`
+
+const AddressCard = styled.div`
+  background: ${(props) => props.theme.colors.background};
+  border: 1px solid ${(props) => props.theme.colors.border};
+  padding-bottom: 2rem;
+  border-radius: 4px;
+  box-shadow: ${(props) => props.theme.colors.tableBoxShadow};
+  margin-bottom: 3rem;
+`
+
+const Address = styled.div`
+    /* background: ${(props) => props.theme.colors.ednBackground};
+    color: ${(props) => props.theme.colors.fail400};
+    padding: 0.5rem; */
+    font-family: "SFMono-Regular", monospace;
+    border-radius: 2px;
+    font-size: 2rem;
+    flex-wrap: wrap;
+    text-transform: uppercase;
+    line-height: 140%;
+    margin-bottom: 1rem;
+`
+
+const CopyButton = styled(ButtonSecondary)`
+  margin-top: 1.5rem;
+`
+
+const CardContainer = styled.div`
+  margin: 2rem;
+  margin-bottom: 0rem;
+`
+
+const Row = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+`
+
+const CardTitle = styled.h2`
+  margin-top: 0rem;
+  margin-bottom: 0rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    margin-bottom: 1rem;
+  }
+`
+
+const Caption = styled.h6`
+  color: ${(props) => props.theme.colors.text200};
+  font-weight: 500;
+`
+
+const SyledCheckbox = styled(Checkbox)`
+  display: flex;
+  min-height: 3rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    min-height: 4rem;
+  }
+`
+
+// TODO confirm/update
```

Comment:
@ryancreatescopy FYI moved the address here & a function to "chunk" it so we only ever define the address in one place.